src/types/timelineTypes.ts

```diff
@@ -54,5 +54,6 @@ export enum ItemTypes {
   AnswerOnAir = 'AnswerOnAir',
   Chapter = 'Chapter',
   Hotspot = 'Hotspot',
-  QuizQuestion = 'QuizQuestion'
+  QuizQuestion = 'QuizQuestion',
+  Summary = 'Summary'
```

Comment:
I don't know if we want to call it "summary" if it might be used for something else later